src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleList } from '../PeopleList';
+import { Loader } from '../Loader';
+
+export const PeoplePage: React.FC = () => {
+  const [persons, setPersons] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople()
+      .then(setPersons)
+      .catch(() => {
+        setIsError(true);
+      })
+      .finally(() => {
+        setIsLoading(false);
+      });
+  }, []);
+
+  const preparedPeople = persons.map(person => {
+    return {
+      ...person,
+      mother: persons.find(mother => person.motherName === mother.name),
+      father: persons.find(father => person.fatherName === father.name),
```

Comment:
At the search stage, you don't know if it's the father or the mother:
```suggestion
      mother: persons.find(({ name })=> person.motherName === name),
      father: persons.find(({ name }) => person.fatherName === name),
```